crates/core/src/tests/integration.rs

```diff
@@ -264,3 +270,239 @@ async fn test_simnet_some_sol_transfers() {
         "Some transfers failed"
     );
 }
+
+
+
+#[tokio::test]
+async fn test_add_alt_fetching() {
+    let mock_tx = VersionedTransaction {
```

Comment:
This is awesome! Could you add in a comment or something indicating what this transaction or doing, or how the `instructions`/`account_keys`/`address_lookup_table` was derived, cause this is quite the intimidating struct 😰